### ISSUE TITLE
Submit billing information to Stripe

### ISSUE THREAD

**Issue by waiting-for-dev** [solidusio-contrib/solidus_stripe#175]
We probably want to push the billing information to Stripe. At this point, we're only sending the user email.

**Comment by loicginoux** [comment]
From my understanding, you attach billing information to a Stripe Customer, if you need to generate invoices or recurrent subscriptions via Stripe. Is it something you want to support in the gem  ? 
[stripe source](https://stripe.com/docs/billing/customer)

**Comment by loicginoux** [comment]
Probably blocked by https://github.com/solidusio/solidus_stripe/issues/163 that needs to be done first

**Comment by waiting-for-dev** [comment]
Thanks for pointing out, @loicginoux. Maybe #163 and this one will be done at once, but let's leave it open so we don't forget.

> generate invoices or recurrent subscriptions via Stripe. Is it something you want to support in the gem ?

That's not part of the initial plan, but probably recurrent subscriptions will be something to consider in integration with [solidus_subscriptions](https://github.com/solidusio/solidus_subscriptions).
